### DISCUSSION TITLE
JSON output?

### DISCUSSION THREAD

**Discussion by matbeedotcom** [jblance/mpp-solar#131] [Q&A]
Currently about to execute mpp-solar binary and read the output, and parse as JSON to gather cell voltages, etc... of my Daly Smart BMS. I'm looking to trigger a relay to start my battery charger, if it gets too low. If I could execute mpp-solar and say I want a JSON response as the output, I would be over the moon happy.

**Comment by jblance** [comment]
Try with `-o json` option added to the command
